print(0)
n = 10
for i in range(n):
    print(i)
raise Exception("An error occurred")